engine/src/flutter/display_list/dl_op_records.h

```diff
@@ -484,8 +484,9 @@ struct TransformResetOp final : TransformClipOpBase {
 
 // 4 byte header + 4 byte common payload packs into minimum 8 bytes
 // DlRect is 16 more bytes, which packs efficiently into 24 bytes total
-// DlRoundRect is 52 more bytes, which packs into 60 bytes total
-// CacheablePath is 128 more bytes, which packs efficiently into 136 bytes total
+// DlRoundRect is 52 more bytes, which packs into 64 bytes total (4 bytes
+// unused). CacheablePath is 128 more bytes, which packs efficiently into 136
```

Comment:
Extremely fine nit: CacheablePath is a different variant of this macro-expanded op struct, so it was broken out into its own line for that reason. Here it makes it look like a concern for the RoundRect variant.

Reply:
Oops! Didn't mean to. Thanks for pointing out.